Kernel/Storage/NVMe/NVMeController.cpp

```diff
@@ -48,7 +48,7 @@ UNMAP_AFTER_INIT ErrorOr<void> NVMeController::initialize()
     m_controller_regs = TRY(Memory::map_typed_writable<volatile ControllerRegister>(PhysicalAddress(m_bar)));
```

Comment:
`adds` -> `add` in the commit message

Reply:
I assume you meant this ` all major OSs such as Linux, FreeBSD adds 1 to the` to ` all major OSs such as Linux, FreeBSD add 1 to the`

Reply:
right

Reply:
Fixed it as well :)